docs/docs/how-tos/sequence.ipynb

```diff
@@ -0,0 +1,327 @@
+    "\n",
+    "</details>\n",
+    "\n",
+    "## Setup\n",
+    "\n",
+    "First, let's install langgraph:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture --no-stderr\n",
+    "%pip install -U langgraph"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"admonition tip\">\n",
+    "     <p class=\"admonition-title\">Set up <a href=\"https://smith.langchain.com\">LangSmith</a> for better debugging</p>\n",
+    "     <p style=\"padding-top: 5px;\">\n",
+    "         Sign up for LangSmith to quickly spot issues and improve the performance of your LangGraph projects. LangSmith lets you use trace data to debug, test, and monitor your LLM aps built with LangGraph — read more about how to get started in the <a href=\"https://docs.smith.langchain.com\">docs</a>. \n",
+    "     </p>\n",
+    " </div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## How to build the graph\n",
+    "\n",
+    "Let's demonstrate a simple usage example. We will create a sequence of three steps:\n",
+    "\n",
+    "1. Populate a value in a key of the state\n",
+    "2. Update the same value\n",
+    "3. Populate a different value\n",
+    "\n",
+    "### Define state\n",
+    "\n",
+    "Let's first define our [state](../../concepts/low_level/#state). This governs the [schema of the graph](../../concepts/low_level/#schema), and can also specify how to apply updates. See [this guide](../../how-tos/state-reducers) for more detail.\n",
+    "\n",
+    "In our case, we will just keep track of two values:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from typing_extensions import TypedDict\n",
+    "\n",
+    "\n",
+    "class State(TypedDict):\n",
+    "    value_1: str\n",
+    "    value_2: int"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define nodes\n",
+    "\n",
+    "Our [nodes](../../concepts/low_level/#nodes) are just Python functions that read our graph's state and make updates to it. The first argument to this function will always be the state:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def step_1(state: State):\n",
+    "    return {\"value_1\": \"a\"}\n",
+    "\n",
+    "\n",
+    "def step_2(state: State):\n",
+    "    current_value_1 = state[\"value_1\"]\n",
+    "    return {\"value_1\": f\"{current_value_1} b\"}\n",
+    "\n",
+    "\n",
+    "def step_3(state: State):\n",
+    "    return {\"value_2\": 10}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that when issuing updates to the state, each node can just specify the value of the key it wishes to update.\n",
```

Comment:
this is actually a super important point and we should highlight it with a callout. there are a lot of people who have an antipattern of returning `{**state, "updated_key": "updated_value}`